docs/Guide.Expo.md

```diff
@@ -0,0 +1,45 @@
+---
+id: Guide.Expo
+title: Expo
+---
+
+## Usage with Expo (iOS)
+
+- Install `detox` and `detox-expo-helpers` (yarn or npm)
+- Add `detox` configuration to [package.json](https://github.com/expo/with-detox-tests/blob/master/package.json#L21-L29):
+
+```es6
+"detox": {
+  "configurations": {
+    "ios.sim": {
+      "binaryPath": "bin/Exponent.app",
+      "type": "ios.simulator",
+      "name": "iPhone 7"
+    }
+  }
+}
+```
+
+- Download the Expo app from [Expo.io/tools](https://expo.io/tools).
+- Unzip the iOS IPA and **rename the folder** to `Exponent.app`. It'll have a file icon but will still be a folder.
+- Create `bin` folder in your project and put `Exponent.app` inside so it matches the binaryPath you set above.
+- Create an `e2e` and copy over the settings from [the example app](https://github.com/expo/with-detox-tests/tree/master/e2e)
+
+The most important piece of this the `reloadApp` from `detox-expo-helpers`. Don't forget this.
+
+```es6
+const { reloadApp } = require('detox-expo-helpers');
+// ...
+beforeEach(async () => {
+  await reloadApp();
```

Comment:
Out of curiosity, is it possible at all to _not_ have to reload the app in between every test? The setup/teardown process adds a fairly significant amount of time to test runs, but when I remove this beforeEach hook from my tests the tests get screwy, so I was wondering if this is just the way it is for now.

Reply:
I think this is just the way it has to be for now.